src/DynamoCore/Core/NodeModelAssemblyLoader.cs

```diff
@@ -198,6 +201,34 @@ internal static bool ContainsNodeModelSubType(Assembly assem)
         {
             return assem.GetTypes().Any(IsNodeSubType);
         }
+        internal static bool ContainsNodeViewCustomizationType(Assembly assem)
+        {
+            return GetCustomizationTypesUsingReflection(assem).Any();
+        }
+
+        internal static IEnumerable<Type> GetCustomizationTypesUsingReflection(Assembly assem)
+        {
+            IEnumerable<Type> output = new Type[] { };
+            //to avoid changing when we load DynamoCoreWpf bail out if it's not yet loaded.
+            if (AppDomain.CurrentDomain.GetAssemblies().All(x => !x.FullName.StartsWith("DynamoCoreWpf"))){
```

Comment:
@pinzart90 this seems to work fine - I'm going to test in Revit context as well just to be sure.